dashboard/app/controllers/pd/application_dashboard_controller.rb

```diff
@@ -0,0 +1,20 @@
+module Pd
+  class ApplicationDashboardController < ApplicationController
+    before_action :authenticate_user!
+
+    def index
+      @permission = nil
+
+      if current_user.permission? UserPermission::WORKSHOP_ADMIN
+        @permission = :workshop_admin
+      end
+
+      unless @permission
+        render_404
+        return
+      end
+
+      view_options(full_width: true)
+    end
+  end
```

Comment:
1. We should ideally be using `CanCan(Can)` authorization here. Do we expect the application dashboard to be available to other types of users with different abilities? In that case it could make sense to do it this way.

 2. Please add some basic unit tests for auth.

Reply:
1. The application dashboard will also be available to regional partners

2. The unit tests will be coming soon in a follow up PR